src/util/FactorUtil.js

```diff
@@ -178,9 +178,19 @@ const getPasswordComplexityRequirementsAsArray = function(policy, i18nKeys) {
 
   if (policy.complexity) {
     const complexityFields = i18nKeys.complexity;
+
     const policyComplexity = setExcludeAttributes(policy.complexity);
+    let filteredPolicyComplexity = policyComplexity;
+
+    // If useADComplexityRequirements is true, ignore casing, number, and symbol rules since
+    // AD validator handles those requirements
+    if (policyComplexity.useADComplexityRequirements) {
+      const allowed = ['minLength', 'useADComplexityRequirements', 'excludeUsername', 'excludeFirstName',
+        'excludeLastName', 'excludeAttributes'];
+      filteredPolicyComplexity = _.pick(policyComplexity, allowed);
+    }
 
-    const requirements = _.map(policyComplexity, function(complexityValue, complexityType) {
+    const requirements = _.map(filteredPolicyComplexity, function(complexityValue, complexityType) {
```

Comment:
Gen 2 changes only require filtering out the complexity fields to render if `useADComplexityRequirements` is present